internal/provider/provider.go

```diff
@@ -453,7 +454,9 @@ func (p *MongodbtlasProvider) Resources(context.Context) []func() resource.Resou
 		streaminstance.Resource,
 		streamconnection.Resource,
 	}
-	previewResources := []func() resource.Resource{} // Resources not yet in GA
+	previewResources := []func() resource.Resource{ // Resources not yet in GA
```

Comment:
Is this a common approach for new resources?

Reply:
no. This resource specifically will be released as a preview resource